content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -429,10 +429,10 @@ generator client {
 
 #### Options
 
-| Option         | Required | Description                                                                                                                   | Default |
-| -------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `--data-proxy` | No       | The `generate` command will generate Prisma Client for use with the [Data Proxy](/data-platform/classic-projects/data-proxy). |
-| `--watch`      | No       | The `generate` command will continue to watch the `schema.prisma` file and re-generate Prisma Client on file changes.         |
+| Option         | Required | Description                                                                                                           | Default |
+| -------------- | -------- | --------------------------------------------------------------------------------------------------------------------- | ------- |
+| `--no-engine`  | No       | The `generate` command will generate Prisma Client for use with [Accelerate](/data-platform/accelerate).              |
```

Comment:
We should:
- Keep both flags since `--data-proxy` is still a valid flag. It would be worth highlighting that we recommend the `--no-engine` flag over `--data-proxy`
- Document the `--accelerate` flag here
- Add a note that `--accelerate is available from version 5.1.0 and `--no-engine` is available from versions 5.2.0 and later

Reply:
Updated! Thank you.